### PR TITLE
Updated package infrastructure to use tox for testing and setuptools_scm

### PR DESCRIPTION
**Pull request by astrofrog**
Updates to the infrastructure to be consistent with other packages

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,3 +58,5 @@ venv
 
 # PyCharm
 .idea
+
+pytest_doctestplus/version.py
```
Comment (pllim) [inline .gitignore:62]:
```suggestion
pytest_doctestplus/version.py
pip-wheel-metadata/
```

**Review by pllim — APPROVED** [review]
Just one minor thing. Otherwise LGTM. Thanks!

**Review by saimn — APPROVED** [review]
Looks good to me too :+1:

**Comment by pllim** [comment]
Thanks!

**Pull request merged**